src/sentry/api/bases/organization.py

```diff
@@ -122,9 +122,6 @@ def convert_args(self, request, organization_slug, *args, **kwargs):
         except Organization.DoesNotExist:
             raise ResourceDoesNotExist
 
-        if organization.status != OrganizationStatus.VISIBLE:
```

Comment:
Does this mean we won't be hiding the org anymore when someone navigates to it when it's pending deletion?

Or is there something else I'm missing that'd change this? Or is this change in behavior intentional? (I know we need to handle it differently to support cancelling, but not sure if these side effects are intentional as well)

Reply:
Yeah we wont hide it. I'd like to expose some information when its in a pending deletion state.